.release-please-manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  ".": "2.6.0",
-  "packages/api-explorer": "0.9.35",
-  "packages/code-editor": "0.1.22",
-  "packages/extension-api-explorer": "22.10.0",
-  "packages/extension-sdk": "22.10.0",
-  "packages/extension-sdk-react": "22.10.0",
-  "packages/extension-utils": "0.1.12",
-  "packages/hackathon": "22.10.0",
-  "packages/run-it": "0.9.35",
-  "packages/sdk": "22.10.0",
-  "packages/sdk-codegen": "21.6.0",
-  "packages/sdk-codegen-scripts": "21.4.1",
+  ".": "2.7.0",
+  "packages/api-explorer": "0.9.36",
+  "packages/code-editor": "0.1.23",
+  "packages/extension-api-explorer": "22.10.1",
+  "packages/extension-sdk": "22.10.1",
+  "packages/extension-sdk-react": "22.10.1",
+  "packages/extension-utils": "0.1.13",
+  "packages/hackathon": "22.10.1",
+  "packages/run-it": "0.9.36",
+  "packages/sdk": "22.11.0",
```

Comment:
This is supposed to be `22.10.1`. What's the right way to fix it? And explicit "release as" in the release config?

Reply:
yes, in the package specific config https://github.com/looker-open-source/sdk-codegen/blob/732e513b3a6f5c937a4f35f35f91b0a2b26dc67f/release-please-config.json#L10

Reply:
There was a feature "feat" which corresponds to a minor bump.
https://github.com/looker-open-source/sdk-codegen/commit/50e925c90eb3c5ffefc8e8535131471e2f2d0334

Should keep in mind that whatever commit title prefix you choose, release-please will automatically set the version:
```
fix: which represents bug fixes, and correlates to a [SemVer](https://semver.org/) patch.
feat: which represents a new feature, and correlates to a SemVer minor.
```
If posisble, its better to set the prefix correctly than override in the config.

Reply:
Thanks for pointing that out. I suppose it could be called a feature. I wonder if changing that description triggers a release-please update. Guess we'll find out now that I've relabeled it as a fix.